libs/island-ui/core/src/lib/Tabs/Tabs.tsx

```diff
@@ -66,7 +66,7 @@ export const Tabs: FC<React.PropsWithChildren<TabInterface>> = ({
   const [isMobile, setIsMobile] = useState(false)
 
   useEffect(() => {
-    if (width < theme.breakpoints.md) {
+    if (width < theme.breakpoints.lg) {
```

Comment:
Maybe we should only do this if the tabs are many? Like 5,6 or so? 🤔  and keep me MD breakpoint otherwise? What do you think?

Reply:
Lets test it out